analyzer_test.go

```diff
@@ -94,78 +98,11 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 
 		when("image exists", func() {
 			it.Before(func() {
-				metadata := h.MustReadFile(t, filepath.Join("testdata", "analyzer", "app_metadata.json"))
+				metadata := h.MustReadFile(t, filepath.Join("testdata", "restorer", "app_metadata.json"))
 				h.AssertNil(t, image.SetLabel("io.buildpacks.lifecycle.metadata", string(metadata)))
 				h.AssertNil(t, json.Unmarshal(metadata, &appImageMetadata))
 			})
 
-			it("restores layer metadata", func() {
-				_, err := analyzer.Analyze(image, testCache)
-				h.AssertNil(t, err)
-
-				for _, data := range []struct{ name, want string }{
-					{"metadata.buildpack/launch.toml", "[metadata]\n  launch-key = \"launch-value\""},
-					{"metadata.buildpack/launch-build-cache.toml", "[metadata]\n  launch-build-cache-key = \"launch-build-cache-value\""},
-					{"metadata.buildpack/launch-cache.toml", "[metadata]\n  launch-cache-key = \"launch-cache-value\""},
-					{"no.cache.buildpack/some-layer.toml", "[metadata]\n  some-layer-key = \"some-layer-value\""},
-				} {
-					got := h.MustReadFile(t, filepath.Join(layerDir, data.name))
-					h.AssertStringContains(t, string(got), data.want)
-				}
-			})
-
-			it("restores layer sha files", func() {
```

Comment:
Same question as https://github.com/buildpacks/lifecycle/pull/505/files#r582333920

Reply:
> restores app and cache layer sha files, prefers app sha
I think that covers this. It was asserting the same files and contents